### PR TITLE
add container image for docker metricsets

### LINKED ISSUE #8214: Add docker.container.image on docker/cpu stats (Metricbeat)
When filtering for `metricset.module:docker AND metricset.name:cpu`, I get a `docker.container.name` but no `docker.container.image` populated.  Could we add it please?

As per @exekias ' comment here: https://discuss.elastic.co/t/no-docker-container-image-on-docker-cpu-stats/147169

Thank you!

### PR DESCRIPTION
**Pull request by pohzipohzi**
this fixes #8214

### CODE REVIEW

**Comment by exekias** [comment]
thank you for taking this @pohzipohzi :tada: :tada: 

It's looking good to me, could you please add a changelog entry?

jenkins, test this please

**Comment by pohzipohzi** [comment]
hey @exekias . Sorry I forgot about that. Here you go

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -79,6 +79,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Add docker diskio stats on Windows. {issue}6815[6815] {pull}8126[8126]
 - Fix incorrect type conversion of average response time in Haproxy dashboards {pull}8404[8404]
 - Fix dropwizard module parsing of metric names. {issue}8365[8365] {pull}6385[8385]
+- Add container image for docker metricsets. {issue}8214[8214] {pull}8438[8438]
```
Comment (exekias) [inline CHANGELOG.asciidoc]: Could you please move it to the added section? This is for bugfixes I beleive

Comment (pohzipohzi) [reply CHANGELOG.asciidoc]: fixed

**Comment by exekias** [comment]
jenkins, test this please

**Comment by exekias** [comment]
Waiting for green :+1: thank you so much for taking this! :tada: :tada: :taco:

**Comment by kierenj** [comment]
thanks @pohzipohzi (I originally requested this) :)

**Pull request merged**